_data/education.yml

```diff
@@ -261,6 +261,17 @@ websites:
       - hardware
     doc: https://kb.mit.edu/confluence/display/istcontrib/Duo+Authentication+Landing+Page
 
+  - name: McGill University
+    url: https://mcgill.ca/
+    img: mcgill.png
+    tfa:
+      - totp
+      - hardware
```

Comment:
What is their hardware 2FA method? I only see hardware token mentioned once under Authentication Step 2: options.

Reply:
I can't find any proper documentation any more, so I removed the hardware entry for now.